dist/web3-light.js

```diff
@@ -3327,6 +3348,145 @@ module.exports = ICAP;
     You should have received a copy of the GNU Lesser General Public License
     along with ethereum.js.  If not, see <http://www.gnu.org/licenses/>.
 */
+/** @file ipcprovider.js
+ * @authors:
+ *   Fabian Vogelsteller <fabian@ethdev.com>
+ * @date 2015
+ */
+
+"use strict";
+
+var utils = require('../utils/utils');
+var errors = require('./errors');
+
+
+var IpcProvider = function (path, net) {
+    var _this = this;
+    this.responseCallbacks = {};
+    this.path = path;
+    
+    net = net || require('net');
```

Comment:
Yes, but in mist its even using a net wrapper package, which tunnels all `net` calls, through the electron backend to filter it over and internal IPC.

So the default net package makes only sense of you're in a node environment. In a browser you always would need to use HTTP or a custom `net` wrapper of some sort.